ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 5.6.1f1
+m_EditorVersion: 2017.1.0f3
```

Comment:
the size wasnt my point. every time you push this file, I get a complaint from unity when I open it. Additionally, I essentially have this file permanently listed as one of my git diffs.